src/types.js

```diff
@@ -23,7 +23,8 @@ export type PersistConfig = {
   stateReconciler?: false | Function,
   getStoredState?: PersistConfig => Promise<PersistedState>, // used for migrations
   debug?: boolean,
-  serialize?: boolean,
+  serialize?: boolean | Function,
+  deserialize?: Function | null,
```

Comment:
Shouldn't `deserialize?` and `serialize?` have the same value here?

Reply:
OK, It should be exactly same.